android/src/com/mapswithme/maps/widget/menu/NavMenu.java

```diff
@@ -50,10 +50,19 @@
 
   private int currentPeekHeight = 0;
 
-  public NavMenu(AppCompatActivity activity, NavMenuListener navMenuListener)
+
+  public interface OnMenuSizeChangedListener
```

Comment:
Why listener? Isn't `mMapFragment.setupBottomWidgetsOffset` a proper place to update the C++ arrow position?